packages/components/unpublished-development-types/global.d.ts

```diff
@@ -25,5 +25,11 @@ declare module '@glint/environment-ember-loose/registry' {
     // we have to use `ember-stargate` version `0.4.3` because version `0.5.0` causes a break in the tests for `ember-lts-3.28`
     Portal: typeof Portal;
     PortalTarget: typeof PortalTarget;
+    'sort-by': HelperLike<{
+      Args: {
+        Positional: [string[], unknown[]];
+      };
+      Return: unknown[];
+    }>;
```

Comment:
question: no luck getting this from `ember-composable-helpers`, right?

Reply:
yeah, unfortunately the library is all JS and I couldn't find any types for it

Reply:
fair